app-kotlin/src/main/res/layout/detail_view_controller.xml

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <include layout="@layout/error_view" />
+
+    <FrameLayout
+        android:id="@+id/detail_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:visibility="gone">
```

Comment:
do we need this nested framelayout?

Reply:
Yes, we do need this to hide the view when the network called failed and an error sign appears.

Reply:
I think you forgot to remove this second frame layout 🤔